OpenStack/RHEL-OSP/7/CMakeLists.txt

```diff
@@ -1 +1,8 @@
 ssg_build_product("rhel-osp7")
+
+if (SSG_CENTOS_DERIVATIVES_ENABLED)
+    ssg_build_derivative_product("rhel-osp7" "centos" "centos-osp7")
+endif()
+if (SSG_SCIENTIFIC_LINUX_DERIVATIVES_ENABLED)
+    ssg_build_derivative_product("rhel-osp7" "sl" "sl-osp7")
+endif()
```

Comment:
@redhatrises We had them in release last release, 0.1.31
See old Makefile, https://github.com/OpenSCAP/scap-security-guide/blob/v0.1.31/OpenStack/RHEL-OSP/7/Makefile#L89
And the zip archive from release pages also contains these data streams.